### ISSUE TITLE
Installation instructions in the readme fail

### ISSUE THREAD

**Issue by reactormonk** [lumihq/purescript-react-basic#125]
> spago install react-basic react-basic-dom react-basic-hooks

gives me

> [error] The following packages do not exist in your package set:
> [error]   - react-basic-dom

**Comment by spicydonuts** [comment]
The package set pr hasn’t been merged yet: https://github.com/purescript/package-sets/pull/652

You can add the packages as they are in that pr to your packages.dhall in the meantime if you’d like to use them before the new package set is released.

**Comment by spicydonuts** [comment]
@reactormonk That command will work now if you update to the latest package set

**Comment by reactormonk** [comment]
Thanks!

**Comment by sezna** [comment]
I'm running into this now. I'm new to PS - I've tried Spago and psc-package on versions 14.7, 14.9, and 15.0. I follow the README instructions and get:
```
psc-package install react-basic
Package `react-basic` does not exist in package set
```
(note the same thing happens with `spago`, just putting `psc-package` here for reference)

my `psc-package.json`:
```
{
  "name": "ps",
  "set": "psc-0.15.0",
  "source": "https://github.com/purescript/package-sets.git",
  "depends": [
    "prelude"
  ]
}

```

Was it removed from the package set?

**Comment by JordanMartinez** [comment]
@sezna We just released PureScript `0.15.0`, and the rest of the ecosystem has yet to finish updating their libraries to the new compiler version. So, I don't think this library has been updated to `0.15.0` yet.
However, I would expect this to work on `0.14.7`. With the `0.14.7` release of `purescript` installed on your computer, could you paste the output of the following sequence of commands?
```sh
mkdir newProj
cd newProj
spago init
spago ls packages | grep 'react'
```

`psc-package` is an older tool that many don't really use anymore. I'd recommend using `spago`.
